database/database_operate.js

```diff
@@ -15,6 +15,19 @@ const product_create = (props, callback) => {
   });
 };
 
+const product_readALL = (callback) => {
+  // create an array of product list
+  const queryStr = 'SELECT * FROM products;';
```

Comment:
THis query will take forever and you will overload javascript's heap memory most likely. You probably don't want to ever run this, it'll tank your virtual machine

Reply:
You are right, I might need to desc maybe only top 50.